.github/workflows/functional-test-cloud.yaml

```diff
@@ -595,10 +595,10 @@ jobs:
                   service-account-private-key-file: /etc/kubernetes/pki/sa.key
           EOF
 
-      - name: Install dapr into cluster
-        run: |
-          wget -q https://raw.githubusercontent.com/dapr/cli/master/install/install.sh -O - | /bin/bash -s ${{ env.DAPR_VER }}
-          dapr init -k --wait --timeout 600 --runtime-version ${{ env.DAPR_VER }} --dashboard-version ${{ env.DAPR_DASHBOARD_VER }}
+      # - name: Install dapr into cluster
+      #   run: |
+      #     wget -q https://raw.githubusercontent.com/dapr/cli/master/install/install.sh -O - | /bin/bash -s ${{ env.DAPR_VER }}
+      #     dapr init -k --wait --timeout 600 --runtime-version ${{ env.DAPR_VER }} --dashboard-version ${{ env.DAPR_DASHBOARD_VER }}
```

Comment:
Hey @superbeeny - I have a question about removing this. 

Does `rad install kubernetes` install fail when Dapr is already installed? I think we should make sure that case succeeds. 

I'm trying to assess whether this is being removed because it's unnecessary, or whether it's being removed because it's **broken**.

Reply:
Rad init will not fail for an existing dapr install